run-test.cmd

```diff
@@ -8,7 +8,7 @@ set LOCATION=%1
 set RUNTIME_PATH=%2
 
 if "%LOCATION%" == "" set LOCATION=%~dp0\bin\Windows_NT.AnyCPU.Debug
-if "%RUNTIME_PATH%" == "" set RUNTIME_PATH=%~dp0\Tools\testdotnetcli
+if "%RUNTIME_PATH%" == "" set RUNTIME_PATH=%~dp0\bin\runtime\netcoreapp-Windows_NT-Debug-x64
```

Comment:
Is that actually the correct path? Shouldn't this be the shared framework for bin\testhost? 

(although I must admit I don't know if we are actually even using this anywhere currently and I want to just delete it completely).